{
	"about": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-017812.html",
	"title": "UPX project の UPX におけるゼロ除算に関する脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-017812.html",
	"description": "UPX project の UPX には、ゼロ除算に関する脆弱性が存在します。",
	"identifier": "JVNDB-2020-017812",
	"references": [
		{
			"text": "https://www.cve.org/CVERecord?id=CVE-2020-27790",
			"source": "CVE",
			"id": "CVE-2020-27790"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2020-27790",
			"source": "NVD",
			"id": "CVE-2020-27790"
		},
		{
			"text": "https://github.com/upx/upx/commit/eb90eab6325d009004ffb155e3e33f22d4d3ca26",
			"source": "関連文書",
			"id": "github.com (eb90eab6325d009004ffb155e3e33f22d4d3ca26)"
		},
		{
			"text": "https://github.com/upx/upx/issues/331",
			"source": "関連文書",
			"id": "github.com (issues/331)"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/369.html",
			"id": "CWE-369",
			"title": "ゼロ除算(CWE-369)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:upx_project:upx",
			"version": "2.2",
			"vendor": "UPX project",
			"product": "UPX"
		}
	],
	"cvss": [
		{
			"version": "3.0",
			"score": "5.5",
			"type": "Base",
			"severity": "Medium",
			"vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
		}
	],
	"date": "2023-09-26T17:29+09:00",
	"issued": "2023-09-26T17:29+09:00",
	"modified": "2023-09-26T17:29+09:00"
}
